galaxyp.yaml

```diff
@@ -178,3 +178,124 @@ tools:
   tool_shed_url: https://toolshed.g2.bx.psu.edu
   revisions:
     - 553f0336e43e
+
+
+
+- name: package_directag
+  owner: galaxyp
+  tool_shed_url: http://toolshed.g2.bx.psu.edu/
+
+- name: package_myrimatch
+  owner: galaxyp
+  tool_shed_url: http://toolshed.g2.bx.psu.edu/
+
+- name: package_mgf_formatter
+  owner: galaxyp
+  tool_shed_url: http://toolshed.g2.bx.psu.edu/
+
+- name: package_ltq_iquant_cli
+  owner: galaxyp
+  tool_shed_url: http://toolshed.g2.bx.psu.edu/
+
+- name: package_idpqonvert
+  owner: galaxyp
+  tool_shed_url: http://toolshed.g2.bx.psu.edu/
+
+- name: package_tagrecon
+  owner: galaxyp
+  tool_shed_url: http://toolshed.g2.bx.psu.edu/
+
+
+
+- name: directag_and_tagrecon
+  owner: galaxyp
+  tool_panel_section_label: Bumbershoot
+  tool_shed_url: http://toolshed.g2.bx.psu.edu/
+
+- name: idpqonvert
+  owner: galaxyp
+  tool_panel_section_label: Bumbershoot
+  tool_shed_url: http://toolshed.g2.bx.psu.edu/
+
+- name: ltq_iquant_cli
+  owner: galaxyp
+  tool_panel_section_id: utils
+  tool_shed_url: http://toolshed.g2.bx.psu.edu/
+
+- name: mgf_formatter
+  owner: galaxyp
+  tool_panel_section_id: utils
+  tool_shed_url: http://toolshed.g2.bx.psu.edu/
+
+- name: msconvert
+  owner: galaxyp
+  tool_panel_section_id: utils
+  tool_shed_url: http://toolshed.g2.bx.psu.edu/
+
+- name: ms_data_converter
+  owner: galaxyp
+  tool_panel_section_id: utils
+  tool_shed_url: http://toolshed.g2.bx.psu.edu/
+
+- name: ms_wiff_loader
+  owner: galaxyp
+  tool_panel_section_id: utils
+  tool_shed_url: http://toolshed.g2.bx.psu.edu/
+
+- name: myrimatch
+  owner: galaxyp
+  tool_panel_section_label: Bumbershoot
+  tool_shed_url: http://toolshed.g2.bx.psu.edu/
+
+- name: peptide_to_gff
+  owner: galaxyp
+  tool_panel_section_id: utils
+  tool_shed_url: http://toolshed.g2.bx.psu.edu/
+
+- name: pepxml_to_xls
+  owner: galaxyp
+  tool_panel_section_id: utils
+  tool_shed_url: http://toolshed.g2.bx.psu.edu/
+
+- name: proteinpilot
+  owner: galaxyp
+  tool_panel_section_id: utils
+  tool_shed_url: http://toolshed.g2.bx.psu.edu/
+
+- name: protxml_to_xls
+  owner: galaxyp
+  tool_panel_section_id: utils
+  tool_shed_url: http://toolshed.g2.bx.psu.edu/
+
+- name: psm_eval
+  owner: galaxyp
+  tool_panel_section_id: utils
+  tool_shed_url: http://toolshed.g2.bx.psu.edu/
+
+
+
+- name: openms
+  owner: galaxyp
+  tool_panel_section_label: OpenMS
+  tool_shed_url: http://testtoolshed.g2.bx.psu.edu/
+
+- name: spectrast
+  owner: iracooke
+  tool_panel_section_label: SpectraST
+  tool_shed_url: http://testtoolshed.g2.bx.psu.edu/
```

Comment:
I guess most of the testtoolshed tools are nowadays already in the MTS. I will check this in the next days.